src/bootstrap-fork.js

```diff
@@ -173,12 +173,13 @@ function terminateWhenParentTerminates() {
 }
 
 function configureCrashReporter() {
-	const crashReporterOptionsRaw = process.env['CRASH_REPORTER_START_OPTIONS'];
-	if (typeof crashReporterOptionsRaw === 'string') {
+	const crashReporterStartOptionsRaw = process.env['CRASH_REPORTER_START_OPTIONS'];
```

Comment:
All the changes in here seem to be just renames, so for the sake of a smaller PR, I would undo them.